KNSB/apirest/src/main/java/com/restapi/apirest/rest/Endpoint.java

```diff
@@ -0,0 +1,108 @@
+package com.restapi.apirest.rest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+//@ApplicationScoped
+@Path("/content")
+public class Endpoint {
+
+    @Inject
+    FileManager manager = new FileManager();
+    @GET
+    @Path("/customer")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCustomer(Customer customer,
```

Comment:
On GET methods you don't have a body in the request. That means this Customer parameter should not be there.